docs/en/siem/machine-learning.asciidoc

```diff
@@ -20,9 +20,9 @@ image::ml-ui.png[]
 [float]
 [[manage-jobs]]
 === Manage machine learning jobs
-For users with the `ml_admin` role, the `Anomaly Detection` interface within
-the main navigation header can be used for for viewing, starting, and stopping
-SIEM machine learning jobs.
+For users with the `machine_learning_admin` role, the `Anomaly Detection`
+interface within the main navigation header can be used for for viewing,
+starting, and stopping SIEM machine learning jobs.
 
 TIP: To add a custom job to the `Anomaly Detection` interface, add a `SIEM` tag 
```

Comment:
This isn't new, but I think that the same feedback as [this comment](https://github.com/elastic/siem-team/issues/519#issuecomment-608583504) applies, where the verb "tag" is potentially misleading.

Reply:
Thanks @rylnd - I'll update it